Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -579,7 +579,7 @@ UpdateFspConfig (
     DEBUG ((DEBUG_ERROR, "Unsupported PCH.\n"));
   }
 
-  if (GetBootMode () != BOOT_ON_FLASH_UPDATE) {
+  if (GetBootMode () != BOOT_ON_FLASH_UPDATE && !PcdGetBool (PcdCmdCorruptShellAppEnabled)) {
```

Comment:
Recommend to remove PcdCmdCorruptShellAppEnabled check. This can lead to security implications if customers enables this flag. Lets add write up in github docs for enabling this tool.